core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorConcurrencyTest.scala

```diff
@@ -295,8 +295,12 @@ class GroupCoordinatorConcurrencyTest extends AbstractCoordinatorConcurrencyTest
     }
   }
 
-  class CommitTxnOffsetsOperation(lock: Option[Lock] = None) extends CommitOffsetsOperation {
-    override def runWithCallback(member: GroupMember, responseCallback: CommitOffsetCallback): Unit = {
+  class CommitTxnOffsetsOperation(lock: Option[Lock] = None) extends GroupOperation[TxnCommitOffsetCallbackParams, TxnCommitOffsetCallback] {
```

Comment:
Changed to subclass `GroupOperation` directly to use the different sets of type parameters `TxnCommitOffsetCallbackParams` and `TxnCommitOffsetCallback` which remains topic-name-based unlike their modified counterparts `CommitOffsetCallbackParams` and `CommitOffsetCallback`. This could be avoided if the signature change is extended to transaction offset commit as well.

Reply:
Are there any downsides of updating the transactional offset commit as well? I can't think of any.

Reply:
Cannot think of one either. Have extended to transaction offset commits in the latest git commit. Now focusing on tests.